### DISCUSSION TITLE
Send older data to influxdb

### DISCUSSION THREAD

**Discussion by SodaSyrup** [alexjustesen/speedtest-tracker#1681] [Q&A]
Hello, is it possible to get the older speedtest data also send to influxdb before i started using it? I saw some complicated ways to try it with a csv, but if there is a way to do it directly with the tracker then id prefer that much more.

**Comment by svenvg93** [comment]
Hi @SodaSyrup , 

Unfortunately this is not possible as the data is only send when a test is executed.
The best would be to import a csv as you mentioned.
